### PR TITLE
feat(sumo): support for SUMO 1.20.0

### PR DESCRIPTION
**Pull request by kschrab**
## Description

* Adds support for SUMO 1.20.0
* Updated version in README
* SUMO 1.20.0 libsumojni.jar is not yet available, therefore currently only upgrade `LibSumoAmbassador` to 1.19.0, but tested already with 1.20.0-SNAPSHOT.
  
## Affected parts of the online documentation

Changes in the documentation required?

Yes, but only on next MOSAIC release.

## Definition of Done

**Prerequisites**

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] Your GitHub user id is linked with your Eclipse Account.

**Required**

- [x] The title of this merge request follows the scheme `type(scope): description`  (in the style of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary))
- [x] You have assigned a suitable label to this pull request (e.g., `enhancement`, or `bugfix`)
- [x] `origin/main` has been merged into your Fork.
- [x] Coding guidelines have been followed (see CONTRIBUTING.md).
- [x] All checks on GitHub pass.
- [x] All tests on [Jenkins](https://ci.eclipse.org/mosaic/job/mosaic/) pass.

**Requested** (can be enforced by maintainers)

- [ ] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [ ] If a bug has been fixed, a new unit test has been written (beforehand) to prove misbehavior
- [ ] There are no new SpotBugs warnings.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -115,7 +115,7 @@
         <version.slf4j>2.0.12</version.slf4j><!-- 2.0.12 is approved #13344 -->
         <version.sqlite-jdbc>3.42.0.0</version.sqlite-jdbc><!-- 3.42.0.0 is approved in #9089 -->
         <!-- note, when upgrading, the field LibSumoAmbassador#VALID_LIBSUMO_VERSIONS needs to be changed too -->
-        <version.libsumo>1.18.0</version.libsumo>
```
Comment (kschrab) [inline pom.xml:118]: SUMO 1.20.0 libsumojni.jar is not yet available, therefore currently only upgrade LibSumoAmbassador to 1.19.0, but tested already with 1.20.0-SNAPSHOT.

Comment (schwepmo) [reply pom.xml:118]: Should we wait until 1.20.0 is available before merging this PR?

**Review by schwepmo — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (kschrab) [reply pom.xml:118]: I asked again!

Comment (kschrab) [reply pom.xml:118]: They added 1.20.0 to the repository already: https://repo.eclipse.org/content/repositories/sumo-releases/org/eclipse/sumo/libsumo/

**Pull request merged**